docs/SLACK_APP.md

```diff
@@ -0,0 +1,17 @@
+### Create new Slack App
+
+To generate a token, you must first create a [new Slack App](https://api.slack.com/apps/new), and add it to your workspace.
```

Comment:
We should really give step-by-step indications here:

1. Go to https://api.slack.com/apps/new
2. Click "Create new app"
3. Give it any app name (e.g: "slackcli")
4. Assign it to the right workspace.

Wait, does that mean that a single app can now only interact with a single workspace? This will cause the `-T/--team` option to be deprecated.